lib/rules/no-danger.js

```diff
@@ -55,13 +56,33 @@ module.exports = {
 
     messages,
 
-    schema: [],
+    schema: [{
+      type: 'object',
+      properties: {
+        customComponentNames: {
+          items: {
+            type: 'string',
+          },
+          minItems: 1,
```

Comment:
```suggestion
          minItems: 0,
```
so that a config can reset the value to the default